pkg/registry/core/pod/strategy.go

```diff
@@ -237,7 +237,7 @@ func PodToSelectableFields(pod *api.Pod) fields.Set {
 	// amount of allocations needed to create the fields.Set. If you add any
 	// field here or the number of object-meta related fields changes, this should
 	// be adjusted.
-	podSpecificFieldsSet := make(fields.Set, 7)
+	podSpecificFieldsSet := make(fields.Set, 8)
```

Comment:
How much difference do you think this would create?

Reply:
I don't think it would solve the regression. But this means that every call to PodsToSelectableFields was doing more than 2x allocations without this change.
But this should be called constant number of times per pod modification hopefully, so probably the difference won't be huge...

Reply:
Sounds reasonable.